README

```diff
@@ -107,14 +107,14 @@ lib/include directory. There are also test programs in the test/
 directory which give examples of use.
 
 Python bindings are in bindings/python and are commented in a similar
-manner. There is also test code in the test/ directory which give
+manner. There is also a test code in the test/ directory which gives
```

Comment:
Just done some more reading and I think I agree, it should be gives as 'code' is third person like you say, good spot!